lib/omniauth/strategies/digitalocean.rb

```diff
@@ -50,6 +54,13 @@ def callback_phase
         super
       end
 
+      #
+      # Retrieves the information about the user
+      #
+      def raw_info
+        @raw_info ||= access_token.get('v2/account')
```

Comment:
This doesn't work as the OAuth base url is different that APIv2's. It tries to reach https://cloud.digitalocean.com/v2/account which 404s. We need something like:

```
  API_BASE_URL = "https://api.digitalocean.com"

  def raw_info
    @raw_info ||= access_token.get("#{API_BASE_URL}/v2/account")
  end
```